### PR TITLE
add crossref to other mathLab packages

### PR DESCRIPTION
**Pull request by twisterbboy**

### CODE REVIEW

**Comment by mtezzele** [comment]
Please move this part at the end of the readme just before the License section. Put it in a new section called Reduced order modelling mathLab's packages. Update the TOC accordingly. Finally this branch needs a rebase against the mathLab:master branch

File: README.md [hunk 1]
```diff
@@ -37,23 +37,15 @@
 	* [Recent works with EZyRB](#recent-works-with-ezyrb)
 * [Authors and contributors](#authors-and-contributors)
 * [How to contribute](#how-to-contribute)
-	* [Submitting a patch](#submitting-a-patch) 
+	* [Submitting a patch](#submitting-a-patch)
+* [Reduced order modelling mathLab's packages](#Reduced-order-modelling-mathLab's-packages)
```
Comment (mtezzele) [inline README.md]: this link does not work. The problem is the capital letter or the quote

File: README.md [hunk 2]
```diff
@@ -221,6 +213,16 @@ just a few small guidelines you need to follow.
 [well-formed commit messages]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
 [pull request]: https://help.github.com/articles/creating-a-pull-request
 
+## Reduced order modelling mathLab's packages
+
+Below you can find a list of interesting and useful other available packages:
```
Comment (mtezzele) [inline README.md]: Below you can find a list of useful reduced order modelling packages from SISSA mathLab group:

File: README.md [hunk 3]
```diff
@@ -37,7 +37,8 @@
 	* [Recent works with EZyRB](#recent-works-with-ezyrb)
 * [Authors and contributors](#authors-and-contributors)
 * [How to contribute](#how-to-contribute)
-	* [Submitting a patch](#submitting-a-patch) 
+	* [Submitting a patch](#submitting-a-patch)
+* [Reduced order modelling mathLab's packages](#reduced-order-modelling-mathLab's-packages)
```
[the remark this answers is not in the document]
Comment (twisterbboy) [reply README.md]: Maybe, it is due to the presence of the apostrophe. If I put &#8217?

**Comment by ndem0** [comment]
The link now works. Please, squash the commits into a single one.

**Pull request merged**